src/Components/Typeahead/Render.purs

```diff
@@ -38,9 +38,7 @@ renderSingle
 renderSingle iprops renderItem renderContainer st =
   HH.div_
     [ Input.inputGroup' HH.div
-      -- [ HE.onClick $ Select.always $ Select.raise $ TA.TriggerFocus unit
```

Comment:
I think this is the part that triggered focus and appeared to remove the selected item when the user clicked on the selection. Not saying you should add it back, but just in case we want to recover that functionality.

Reply:
Yea, that line was originally re-triggering the typeahead, but it was causing a UI bug where you couldn't close the typeahead list unless you first clicked into the textbox first. The new version of select has taken a lot more of the responsibility of triggering focus into its own hands - I believe thats why its acting a little differently now.